src/caselawclient/xquery/get_judgment.xqy

```diff
@@ -14,19 +14,23 @@ declare variable $search_query as xs:string? external;
 let $number_marks_xslt := (
   <xsl:stylesheet xmlns:xsl="http://www.w3.org/1999/XSL/Transform"
                   xmlns:uk="https://caselaw.nationalarchives.gov.uk/akn"
+                  xmlns:akn="http://docs.oasis-open.org/legaldocml/ns/akn/3.0"
                   version="2.0">
     <xsl:output method="xml" />
     <xsl:template match="@*|node()">
       <xsl:copy>
         <xsl:apply-templates select="@*|node()"/>
       </xsl:copy>
     </xsl:template>
+    <xsl:template match="//akn:meta//uk:mark">
+          <xsl:apply-templates />
+    </xsl:template>
     <xsl:template match="uk:mark">
       <xsl:copy>
           <xsl:copy-of select="@*" />
           <xsl:attribute name="id">
               <xsl:text>mark_</xsl:text>
-              <xsl:value-of select="count(preceding::uk:mark)"/>
+              <xsl:number count="//uk:mark" level="any" from="//*[ancestor::akn:meta]" />
```

Comment:
Whilst I accept that this is entirely correct in what it's doing, it makes me sad that AkN doesn't just have a `<body>` section or similar we could scope this to and be done with it!

Reply:
Yep.